vars/Ubuntu-1804.yml

```diff
@@ -0,0 +1,5 @@
+---
+__unattended_allowed_origins:
+  - '${distro_id}:${distro_codename}'
```

Comment:
Just one question on this part!

For Ubuntu 16, there is only `-security` currently if you use the role, by default:

https://github.com/jnv/ansible-role-unattended-upgrades/blob/909b3456af7515647397611d9e832c39ae3103d4/vars/Ubuntu.yml#L1-L3

If I understand well, the line above is introducing a change in default behaviour, is that correct @VoidAndAny?

Reply:
Ok I found [more information](https://www.linuxbabe.com/ubuntu/automatic-security-update-unattended-upgrades-ubuntu-18-04) which seems to indicate that:

> The first origin "${distro_id}:${distro_codename}" is necessary because security updates may pull in new dependencies from non-security sources. This origin doesn’t provide software updates.

So this makes sense! If anyone with more expertise on that part can comment, it's still welcome :smile: